pkg/vulnsrc/vulnsrc.go

```diff
@@ -137,60 +137,85 @@ type Optimizer interface {
 }
 
 type fullOptimizer struct {
-	dbc db.Operation
+	dbConfig db.VulnOperation
```

Comment:
Where is this used?

Reply:
https://github.com/aquasecurity/trivy-db/blob/c47c05449be94fe880be11e46c0f6aed18631e26/pkg/vulnsrc/vulnsrc_test.go#L415

Reply:
I found it in the test code, but I couldn't find it in vulnsrc.go. If it is not called in vulnsrc.go, why is this necessary?

Reply:
I found it in the test code, but I couldn't find it in vulnsrc.go. If it is not called in business logic, why is this necessary?

Reply:
Ah I overlooked. I had initially intended it for being able to inject different behaviours for `GetVulnerabilityDetail` but I never got around finishing it and now it's not needed anymore since I added tests for optimizers on their own with different behaviours of `GetDetailFunc`. I have removed this for now https://github.com/aquasecurity/trivy-db/pull/32/commits/f9156e305e100444602cc79ed5930dee879517fe